ceph/rados/core_workflows.py

```diff
@@ -4198,16 +4191,19 @@ def get_osd_status(self, osd_id):
         base_cmd = f"ceph tell osd.{osd_id} status"
         return self.run_ceph_command(cmd=base_cmd)
 
-    def get_osd_perf_dump(self, osd_id):
+    def get_osd_perf_dump(self, osd_id, filter=None):
         """
         method to get the perf dump of osd-
         ceph tell osd.N perf dump command
         Args:
             osd_id: ID of desired osd daemon
+            filter: perf dump filter
```

Comment:
Add some examples of allowed filter keywords in doctext

Reply:
Examples added